test/unit/lib/redmine/export/pdf/issues_pdf_test.rb

```diff
@@ -18,8 +18,9 @@
 require File.expand_path('../../../../../../test_helper', __FILE__)
 
 class IssuesPdfHelperTest < ActiveSupport::TestCase
+  
   #fixtures :users, :projects, :roles, :members, :member_roles,
```

Comment:
Regarding the code, Samples data class needed for executing the test. Try to add fixtures class missing

Reply:
Take a look at the last commit i added some fixtures but the test case failed